model/src/w3tidemd.F90

```diff
@@ -1,3 +1,6 @@
+!> @file w3tidemd.F90
+!> @brief Contains MODULE W3TIDEMD for tidal analysis of time series.
```

Comment:
We put an author at the file level in all cases, as well as an author for all functions and subroutines. Sometimes this seems redundant, but they are different things, and doxygen shows them in different places...

Reply:
I know exactly what you're saying.  Good point, I'll make the change.